include/SFML/System/Vector2.inl

```diff
@@ -159,3 +291,18 @@ constexpr bool operator !=(const Vector2<T>& left, const Vector2<T>& right)
 {
     return (left.x != right.x) || (left.y != right.y);
 }
+
+
+////////////////////////////////////////////////////////////
+// Static member data
+////////////////////////////////////////////////////////////
+
+// Note: the 'inline' keyword here is technically not required, but VS2019 fails
+// to compile with a bogus "multiple definition" error if not explicitly used.
+template <typename T>
+inline constexpr Vector2<T> Vector2<T>::UnitX(static_cast<T>(1), static_cast<T>(0));
+
+// Note: the 'inline' keyword here is technically not required, but VS2019 fails
+// to compile with a bogus "multiple definition" error if not explicitly used.
```

Comment:
For consistency with other files, we only repeat this comment once at the beginning, not in every `inline constexpr` variable.

Furthermore, `inline` might not be required here as it is implied by `template`, maybe VS2017 will be happy with it without `inline`... give it a try.

Reply:
Seems to work without `inline` in VS 2019.